config/jobs-chromeos.yaml

```diff
@@ -0,0 +1,62 @@
+_anchors:
+
+  kbuild-gcc-10-arm64-chromeos: &kbuild-gcc-10-arm64-chromeos-job
+    template: kbuild.jinja2
+    kind: kbuild
+    image: kernelci/staging-gcc-10:arm64-kselftest-kernelci
+    params: &kbuild-gcc-10-arm64-chromeos-params
+      arch: arm64
+      compiler: gcc-10
+      cross_compile: 'aarch64-linux-gnu-'
+      cross_compile_compat: 'arm-linux-gnueabihf-'
+      defconfig: 'cros://chromeos-{krev}/{arch}/chromiumos-{arch}-generic.flavour.config'
+      fragments:
+        - arm64-chromebook
+        - CONFIG_MODULE_COMPRESS=n
+
+  kbuild-gcc-10-x86-chromeos: &kbuild-gcc-10-x86-chromeos-job
+    <<: *kbuild-gcc-10-arm64-chromeos-job
+    image: kernelci/staging-gcc-10:x86-kselftest-kernelci
+    params: &kbuild-gcc-10-x86-chromeos-params
+      arch: x86_64
+      compiler: gcc-10
+      defconfig: 'cros://chromeos-{krev}/{arch}/chromiumos-{arch}-generic.flavour.config'
+      fragments:
+        - x86-board
+        - CONFIG_MODULE_COMPRESS=n
+
+jobs:
+
+  baseline-arm64-chromeos-mediatek: &baseline-job
+    template: baseline.jinja2
+    kind: test
+
+  baseline-arm64-chromeos-qualcomm: *baseline-job
```

Comment:
I might be missing something: what is the purpose of duplicating job definitions with the same body? Is it for logging/tracking progress? Wouldn't adding `job: baseline-job` in the schedule item definition cut it?

Reply:
That's a good question, maybe @nuclearcat knows?

I confess I kept things as they were previously (1:1 mapping between jobs and scheduler entries), but I'd be happy to drop duplicates if we can actually schedule multiple instances of a given job.